default/templates/default/to-json.html

```diff
@@ -10,13 +10,16 @@
 
 {% block body %}
 <div class="response-success alert alert-info hidden">
-    <b>{% trans "Success!" %}</b>
-    <ul>
-        <li>
-            result.zip (<span class="file-size-json"></span>)
-            <a href="javascript:void(0)" class="download-json">{% trans "Download" %}</a>
-        </li>
-    </ul>
+    <b>{% trans "Success!" %}</b> result.zip(<span class="file-size-json"></span>)
+    <a href="javascript:void(0)" class="download-json">{% trans "Download" %}</a>
```

Comment:
I think you should put the form div into a snippet. Use an array to hold the options available for each tool, and a `for` loop to render the options.